arangod/Pregel/WorkerInterface.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Pregel/WorkerConductorMessages.h"
+
+#include <Futures/Future.h>
+
+namespace arangodb::pregel {
+
+class NewIWorker {
```

Comment:
Why "new"?

Reply:
This Worker interface is only temporary because I changed only one function - loadGraph - to use futures: Currently we still have the IWorker interface we need for the Worker. After I converted all functions in IWorker to futures, we get rid of IWorker and only use NewIWorker (which I will then rename to IWorker).